cms/server/admin/templates/participation_preview.html

```diff
@@ -0,0 +1,53 @@
+{% extends base.html %}
+
+{% block core %}
+
+<h1>Import users</h1>
+
+<div id="something">
+  <form enctype="multipart/form-data" action="{{ url_root }}/contest/{{ contest.id }}/users/import" method="POST">
+    <input type="hidden" name="action" value="save">
+    <table class="bordered">
+      <thead>
+        <tr>
+          <th>New</th>
```

Comment:
Now that I think about it, users preview does not have this column, and participants preview does. I suggest to do the same (add or remove) for both preview pages.